tests/cli/test_suite.py

```diff
@@ -1281,138 +2165,419 @@ def test_suite_scaffold_on_existing_suite_raises_error(
     )
 
 
-@pytest.mark.xfail(
-    reason="This command is not yet implemented for the modern API",
-    run=True,
-    strict=True,
-)
 @mock.patch(
     "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
 )
 @mock.patch("subprocess.call", return_value=True, side_effect=None)
-def test_suite_scaffold_creates_notebook_and_opens_jupyter(
-    mock_subprocess, mock_emit, caplog, monkeypatch, titanic_data_context_stats_enabled
+@mock.patch("webbrowser.open", return_value=True, side_effect=None)
+def test_suite_new_profile_creates_notebook_no_jupyter(
```

Comment:
How about changing the name of this test since it runs the notebook and generates a suite?

Reply:
Done -- in the next commit.